strings/aho_corasick.py

```diff
@@ -12,13 +13,13 @@ def __init__(self, keywords):
             self.add_keyword(keyword)
         self.set_fail_transitions()
 
-    def find_next_state(self, current_state, char):
+    def find_next_state(self, current_state: int, char: str) -> Optional[List]:
```

Comment:
```suggestion
    def find_next_state(self, current_state: int, char: str) -> Union[int, None]:
```
Use `Union` for multiple return types and it doesn't even return a list.

Reply:
My bad